make/jdk/src/classes/build/tools/cldrconverter/CLDRConverter.java

```diff
@@ -229,6 +231,12 @@ public static void main(String[] args) throws Exception {
                         tzDataDir = args[++i];
                         break;
 
+                    case "-jdk-header-template":
+                        jdkHeaderTemplate = new String(
+                            Files.readAllBytes(Paths.get(args[++i])),
```

Comment:
Can `java.nio.file.Files#readString(java.nio.file.Path)` be used instead?

Reply:
Ah yes, I was looking for that method and didn't find it, but I now realize I was looking in the jdk 8 api. Punished for not setting up an IDE for editing these files. Fixed now, thanks!